docs/releasenotes/index.rst

```diff
@@ -9,22 +9,19 @@ You can learn more about where we might be going with future releases by:
 * Stopping by the ``#clusterhq`` channel on ``irc.freenode.net``.
 * Visiting our GitHub repository at https://github.com/ClusterHQ/flocker.
 
-Next Release
-============
-
-* The Control Service API enforces that the type of a dataset ``maximum_size`` is an integer, rather then any number.
-
 This Release
 ============
 
-v1.10.1
+v1.10.2
 -------
 
-* The :ref:`Flocker documentation <supported-orchestration-frameworks>` has been re-designed to better reflect that Flocker now integrates with Cluster Managers, rather than providing its own container management features.
-* The new :ref:`CloudFormation installer <cloudformation>` has been made available, to provide a far simpler installation experience for users on AWS.
-* The :ref:`Flocker plugin for Docker <plugin>` should support the direct volume listing and inspection functionality added to Docker 1.10.
-* Fixed a regression that caused block device agents to poll backend APIs like EBS too frequently in some circumstances.
-* Increase limit on maximum Flocker volumes per AWS instance from 11 to 21.
+* Decreased the CPU usage of the Control Service API under load.
+   * The Control Service API enforces that the type of a dataset ``maximum_size`` is an integer, rather then any number.
```

Comment:
It might sound better to say "The Control Service API enforces a dataset's `maximum_size` to be an integer, rather than any number type."